Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -789,7 +789,7 @@ final class PackagePIFProjectBuilder: PIFProjectBuilder {
             return nil
         }
 
-        let bundleName = "\(package.manifestName)_\(target.name)" // TODO: use identity instead?
+        let bundleName = "\(package.manifest.displayName)_\(target.name)" // TODO: use identity instead?
```

Comment:
@neonichu @abertelrud I would have liked to use identity here but not sure about the downstream effects. wdyt?

Reply:
We should do the same as for natively built bundles, which is to mangle the identity to get rid of path separator.

Reply:
see above re. mangling consideration